### ISSUE TITLE
Alpine Linux build issue

### ISSUE THREAD

**Issue by TBK** [ohwgiles/laminar#94]
Hi, I am trying to package laminar on Alpine Linux.

Unfortunately I hit a couple of issues.

**1. Type issue**  - [buildlog.txt](https://github.com/ohwgiles/laminar/files/3217092/buildlog.txt)
```
[ 36%] Building CXX object CMakeFiles/laminard.dir/src/database.cpp.o
/home/tbk/Projects/aports/testing/laminar/src/laminar-0.7/src/client.cpp:70:47: error: 'uint' has not been declared
 static void printTriggerLink(const char* job, uint run) {
                                               ^~~~
In file included from /home/tbk/Projects/aports/testing/laminar/src/laminar-0.7/src/database.cpp:19:
/home/tbk/Projects/aports/testing/laminar/src/laminar-0.7/src/database.h:122:31: error: 'uint' has not been declared
         void bindValue(int i, uint e);
                               ^~~~
/home/tbk/Projects/aports/testing/laminar/src/laminar-0.7/src/database.h:122:14: error: 'void Database::Statement::bindValue(int, int)' cannot be overloaded with 'void Database::Statement::bindValue(int, int)'
         void bindValue(int i, uint e);
              ^~~~~~~~~
/home/tbk/Projects/aports/testing/laminar/src/laminar-0.7/src/database.h:121:14: note: previous declaration 'void Database::Statement::bindValue(int, int)'
         void bindValue(int i, int e);
              ^~~~~~~~~
/home/tbk/Projects/aports/testing/laminar/src/laminar-0.7/src/database.h:151:12: error: 'uint' does not name a type; did you mean 'int'?
 template<> uint Database::Statement::fetchColumn(int col);
            ^~~~
            int
/home/tbk/Projects/aports/testing/laminar/src/laminar-0.7/src/database.h:153:12: error: 'ulong' does not name a type; did you mean 'long'?
 template<> ulong Database::Statement::fetchColumn(int col);
            ^~~~~
            long
/home/tbk/Projects/aports/testing/laminar/src/laminar-0.7/src/database.cpp:84:44: error: 'uint' has not been declared
 void Database::Statement::bindValue(int i, uint e) {
```

I fixed the type issue with these changes - https://github.com/ohwgiles/laminar/pull/93

**2. Failing tests** - [buildlog2.txt](https://github.com/ohwgiles/laminar/files/3217120/buildlog2.txt)
```
[ RUN      ] RunTest.WorkingDirectory
/home/tbk/Projects/aports/testing/laminar/src/laminar-0.7/test/test-run.cpp:81: Failure
Expected equality of these values:
  cwd + "\n"
    Which is: "/tmp/laminar-test-DglBDl/run/foo/1\n"
  readAllOutput()
    Which is: "foo.run: applet not found\n"
[  FAILED  ] RunTest.WorkingDirectory (1 ms)
[ RUN      ] RunTest.SuccessStatus
```

```
[----------] Global test environment tear-down
[==========] 19 tests from 5 test cases ran. (11 ms total)
[  PASSED  ] 15 tests.
[  FAILED  ] 4 tests, listed below:
[  FAILED  ] RunTest.WorkingDirectory
[  FAILED  ] RunTest.SuccessStatus
[  FAILED  ] RunTest.Environment
[  FAILED  ] RunTest.ParamsToEnv
```

**Comment by sroracle** [comment]
The second issue is because that part of laminar's test suite relies on the creation of symlinks to common commands (`pwd`, `true`, `false`, `env`, `yes`) to act as the `.run` file for a job. This doesn't work with `busybox` however, since `busybox` will only see the symlink name (e.g. `/var/lib/laminar/cfg/jobs/WorkingDirectory.run`) instead of a valid applet filename (e.g. `pwd`). I have opened a separate PR for this issue at #95.

**Comment by ohwgiles** [comment]
Solved by 7b418a5 and 1642899
